### PR TITLE
missing styled batch methods for rects and arcs

### PR DESCRIPTION
**Pull request by DerThorsten**
* missing styled methods:
    * `fill_styled_rects` /  `stroke_styled_rects`
    * `fill_styled_arcs` /  `stroke_styled_arcs`
* updated example
* auto creating dict of commands from list of commands to avoid hand-enumerating of commands on python side

### CODE REVIEW

File: ipycanvas/canvas.py [hunk 1]
```diff
@@ -20,20 +20,20 @@
...
+             'rect', 'arc', 'ellipse', 'arcTo', 'quadraticCurveTo',
+             'bezierCurveTo', 'fillText', 'strokeText', 'setLineDash', 'drawImage',
+             'putImageData', 'clip', 'save', 'restore', 'translate',
+             'rotate', 'scale', 'transform', 'setTransform', 'resetTransform',
+             'set', 'clear', 'sleep', 'fillPolygon', 'strokePolygon',
+             'strokeLines', 'fillPolygons', 'strokePolygons', 'strokeLineSegments',
+             'fillStyledRects', 'strokeStyledRects', 'fillStyledCircles', 'strokeStyledCircles',
+             'fillStyledArcs', 'strokeStyledArcs', 'fillStyledPolygons',
+             'strokeStyledPolygons', 'strokeStyledLineSegments']
+COMMANDS = {v: i for i, v in enumerate(_CMD_LIST)}
```
Comment (martinRenou) [inline ipycanvas/canvas.py:36]: Thks!

**Comment by DerThorsten** [comment]
@martinRenou from my side this would be ready to merge =)

What would be missing for a new ipycanvas release? I guess some updated documentation. Any thing else missing? My box2d (now at https://github.com/pyb2d/pyb2d/) has landed on conda forge and I would love to have a binder example with the jupyter integration. 
But for that I would need this  PR here to merged and released.

**Comment by martinRenou** [comment]
Yes, it's missing documentation.

I'll push a PR to try and fix the CI.

**Comment by martinRenou** [comment]
Thanks!

**Pull request merged**